mscore/uploadscoredialog.ui

```diff
@@ -129,6 +126,9 @@
          <property name="locale">
           <locale language="English" country="UnitedStates"/>
          </property>
+         <property name="sizeAdjustPolicy">
+          <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>

```

Comment:
Maybe this and the other could get set programatically, only if the screen is not high enough?
And this here and or the other (or the other only), only if the other is also visible, i.e. on updating an existing score?

Actually I've tried it out (it is possible programatically): both settings are needed on screens less than 786 pixels high, only the other setting on screens less than 800 pixels high.